### ISSUE TITLE
Error: cannot enable long stack traces after promises have been created.

### ISSUE THREAD

**Issue by gersongoulart** [brillout/gulp-jspm#37]
@brillout thank you for `gulp-jspm`. I've been using it for a little while and has been great so far!

Today I updated all packages in my complex build system where some of the gulp plugins I use also depend on Bluebird. Turns out the build is breaking in this line: https://github.com/brillout/gulp-jspm/blob/master/index.js#L6

Apparently `Promise.longStackTraces` affects Bluebird globally, and since promises have already been instantiated in earlier parts of the build, I get this error:

```Error: cannot enable long stack traces after promises have been created. See http://goo.gl/MqrFmX```

Perhaps the `Promise.longStackTraces();` call could be wrapped in a try/catch block to avoid conflicts? (Doing that locally suppressed the error while allowing my build to complete successfully even if without long stack traces).

**Comment by brillout** [comment]
Actually a Library shouldn't set `longStackTraces` but I'm not sure how to then give the user the possibility to enable it and I'm not sure if the performance penalty is that bad to justify having the user to explicitly enable it. But that's another topic.

As for your issue; would using `Promise.config` help? See http://bluebirdjs.com/docs/api/promise.config.html.
I don't know if it also throws an exception if called another time. The doc doesn't seem to mention this.

**Comment by gersongoulart** [comment]
Yes, I also tried the config solution and it did not work either (I got the exact same error, hence I believe either method does exactly the same to Bluebird internally).

I suggested the use of try/catch because that would keep longStackTraces around unless there's another Library already using Bluebird's Promise. Allowing the user to set that config manually wouldn't help in this situation.

**Comment by gersongoulart** [comment]
I'm temporarily publishing the npm package `gulp-jspm-fix` to avoid having our whole team unable to build our app... Let me know if you'd like me to send out a PR with this simple solution (removing longStackTraces).

**Comment by brillout** [comment]
Yea, a PR with the try catch solution would be awesome! 👍

**Comment by gersongoulart** [comment]
@brillout I'm sorry it took me so long to get back to this... I somehow manage to loose track of the issue :P
